src/Smartstore/Threading/AsyncLock.cs

```diff
@@ -80,9 +86,16 @@ public Task ReleaseAsync()
 
             public void Release()
```

Comment:
Something is wrong here. Please check again.

Reply:
Can you please be more specific?

Reply:
Can you check now please?